src/edge/edgeManager.ts

```diff
@@ -424,7 +436,7 @@ export class EdgeManager {
                 break;
             case Constants.LANGUAGE_C:
                 await new Promise((resolve, reject) => {
-                    download("github:Azure/azure-iot-edge-c-module#master", path.join(parent, name), (err) => {
+                    download(`github:Azure/azure-iot-edge-c-module#${Versions.cTemplateVersion()}`, path.join(parent, name), (err) => {

```

Comment:
Similar issue with Python template. We should explicitly fall back to master when cTemplateVersion is null.